src/video/video_sdl2.c

```diff
@@ -154,6 +159,9 @@ static void Video_Mouse_Move(uint16 x, uint16 y)
 	if (s_mouseMinY != 0 && ry < s_mouseMinY) ry = s_mouseMinY;
 	if (s_mouseMaxY != 0 && ry > s_mouseMaxY) ry = s_mouseMaxY;
 
+	if (ry > SCREEN_HEIGHT * g_screen_magnification - 1) ry = SCREEN_HEIGHT * g_screen_magnification - 1;
+	if (rx > SCREEN_WIDTH * g_screen_magnification - 1) rx = SCREEN_WIDTH * g_screen_magnification - 1;
+
```

Comment:
This checks mouse x/y limits to bottom and right, for fake Fullscreeen_Desktop mode.